config/config.js

```diff
@@ -55,8 +54,7 @@ var config = {
             "spreadsheet": {
```

Comment:
I wont ask you to do this, but we should be able to drop the entire spreadsheet section of the config - as soon as we remove the dependencies on src/spreadsheets.js